mapie/utils.py

```diff
@@ -171,6 +171,9 @@ def check_cv(
     ValueError
         If the cross-validator is not valid.
     """
+    if random_state is None:
```

Comment:
You need to add `None` to one of the possible options. In the `raise ValueError()`.

Reply:
Ah no no. All good 😀